documentation/Taiko Fee Proposal.md

```
@@ -0,0 +1,173 @@
+# Taiko Fee Proposal
+
+## Overview
+- I have an intuition that some of the L2 fee-pricing challenges are downstream of reusing and modifying the L1 pricing model, which may not be aligned with the economics of L2s.
+- In particular, it seems to me that L1 protocol-enforced-fees are useful to:
+    - apply a "negative externality fee" to account for costs to the network. This accounts for the cost of running the consensus mechanism.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)